samples/algorithms/integer-factorization/Untitled.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Should this be called "Untitled.ipynb"? I'd suggest a more specific name here.

Reply:
Good catch, I renamed it `Shor.ipynb` for consistency, and added to the README file list.